cmd/vic-machine/inspect/inspect.go

```diff
@@ -89,6 +97,17 @@ func (i *Inspect) Flags() []cli.Flag {
 	return flags
 }
 
+func (i *Inspect) ConfigFlags() []cli.Flag {
+	output := cli.StringFlag{
+		Name:        "output",
+		Value:       "",
+		Usage:       "Determine the format of configuration output",
```

Comment:
feels like user will need to know what is accepted value for this option. So far, it's only "raw".